.coveragerc

```diff
@@ -0,0 +1,7 @@
+[run]
```

Comment:
Thanks for including this! I realized coverage was messed up because I had previously included this information in the `pyproject.toml`, but we aren't using that format for this repo 🙂 .

Reply:
Please double check the nox/integration setup (including the `precommit` setup). I have made changes to those but am not confident if I did them 100% correctly.

Reply:
:+1: will do!